lib/msal-browser/src/utils/BrowserConstants.ts

```diff
@@ -13,6 +13,8 @@ export const BrowserConstants = {
     CACHE_LOCATION_LOCAL: "localStorage",
     // Session storage constant string
     CACHE_LOCATION_SESSION: "sessionStorage",
+    // In memory storage string
+    CACHE_LOCATION_IN_MEMORY: "inMemoryStorage",
```

Comment:
```suggestion
    CACHE_LOCATION_IN_MEMORY: "memoryStorage",
```

Reply:
And can we create an enum for these strings, and export it?

Reply:
I can have Node expose this too if this is an exported enum and moved to Common!

Reply:
I think these are just browser considerations that are not relevant for Node.

Reply:
done.

Reply:
Node will have this relevant when we move out of `serializedCache` but it isn't needed now.